### ISSUE TITLE
Reproducing the result

### ISSUE THREAD

**Issue by liuyuzhenn** [sfu-gruvi-3dv/msp_rot_avg#2]
Hello! Could you provide an instruction on running the code and reproducing the result of the paper?

**Comment by corsy** [comment]
Please clone the updated repository: git clone git@github.com:sfu-gruvi-3dv/msp_rot_avg.git

Then download the trained model with 2 samples from YFCC100 dataset. 
https://drive.google.com/file/d/1p5hq-Y1h3KZunD1mEQ8tlJbqDjfofv3q/view?usp=sharing

Extract folder and put it into the code directory: ./msp_rot_avg/models

The trained model includes vlad and trained msp + finenet. For each yfcc sample:

•	lmdb file: cached edge and node feature as the input of AGF
•	The sampled sub-graphs are cached in 80node_valid_yfcc_<name>.bin
•	Bundle.out, covis.npy, raw_image.list.txt are reconstructed scene with graphs
•	edge_rt_cache.bin: relative rotation and normalized translation of edges

To setup the environment, please refer to the install_pkg.sh. As those packages are required:

•	python-lmdb, torch-scatter, torch-sparse, torch-cluster, torch-geometric, dgl and torchgeometry.
•	We use the pytorch 1.7 in our experiments.

To run the sample (e.g. big_ben_2)

•	cd msp_rot_avg & export PYTHONPATH=$PWD
•	python ./pipeline/rot_avg_mspt.py –dataset . /test_config/yfcc_80nodes_big_ben_2.json

I have tested the script on a TITAN 12G GPU with CUDA11.2, It should look like this:

![pic](https://user-images.githubusercontent.com/10144510/216756364-f87d5d9e-9096-4c33-945b-9b2420d281f5.png)


I will send the rest of the yfcc dataset if needed, whats your email address?

Thanks,
Luwei

**Comment by liuyuzhenn** [comment]
Hello! Could you please provide the whole dataset of YFCC100 dataset? It would be very helpful to my work.

**Comment by corsy** [comment]
Hi Yuzhen, can you send a mail to luweiy@sfu.ca? The original dataset contains copyright and I can send you via mail.
